examples/Hermitian_Phasing.py

```diff
@@ -23,27 +23,27 @@
 t2, td, vd, power, ph1, ph2 = s.symbols("t2 td vd power ph1 ph2")
 echo_time = 10e-3
 f_range = (-400, 400)
-
+selection_range = (None,0.03) #2 x time domain signal
 with figlist_var() as fl:
     for expression, orderedDict, signal_pathway, indirect, label in [
         (
             (
                 23
                 * (1 - 2 * s.exp(-vd / 0.2))
                 * s.exp(+1j * 2 * s.pi * 100 * t2 - abs(t2) * 50 * s.pi)
-            ),
-            [
+            ),    
+            [    
                 ("vd", nddata(r_[0:1:40j], "vd")),
                 ("ph1", nddata(r_[0:4] / 4.0, "ph1")),
                 ("ph2", nddata(r_[0, 2] / 4.0, "ph2")),
                 ("t2", nddata(r_[0:0.2:256j] - echo_time, "t2")),
-            ],
-            {"ph1": 0, "ph2": 1},
+            ],    
+            {"ph1": 0, "ph2": 1},    
             "vd",
             "IR",
-        ),
-        (
-            (
+        ),    
+        (    
+            (    
```

Comment:
it's showing that you added spaces at the end of the line.  Just get rid of them so it matches.